packages/@okta/vuepress-site/guides/sign-into-spa/section01/index.md

```diff
@@ -1,16 +1,19 @@
 ---
-title: Getting Started
+title: Sign Users into Your Single-Page App
 ---
 
+# Sign Users into Your Single-Page App
+
 This guide walks you through how to sign users into your single-page app (SPA) and supports apps created using:
 
 * Angular
 * React
 * Vue
 
-If you are building a web app that is served by a server framework, see [Sign Users into Your Web App](sign-users-into-your-web-applink). If you are building a mobile app, see [Sign Users into Your Mobile App](sign-users-into-your-mobile-applink).
+If you are building a web app that is served by a server framework, see [Sign Users into Your Web App]. If you are building a mobile app, see [Sign Users into Your Mobile App].
```

Comment:
If we are talking about going live with just the SPA guide, we don't want to include links that don't work, right?

Reply:
(at first)